vault/rollback.go

```diff
@@ -101,12 +106,17 @@ func (m *RollbackManager) run() {
 
 // triggerRollbacks is used to trigger the rollbacks across all the backends
 func (m *RollbackManager) triggerRollbacks() {
-	m.mounts.RLock()
-	defer m.mounts.RUnlock()
 	m.inflightLock.Lock()
 	defer m.inflightLock.Unlock()
 
-	for _, e := range m.mounts.Entries {
+	mounts, mountsLock := m.mounts()
+	mountsLock.RLock()
+	defer mountsLock.RUnlock()
+	if mounts == nil {
+		return
+	}
+
+	for _, e := range mounts.Entries {
```

Comment:
I think we still have the issue of paths that get unmounted are still around or no?